### ISSUE TITLE
Unable to install scoped packages, e.g. @angular/common

### ISSUE THREAD

**Issue by ziacik** [verdaccio/verdaccio#55]
`npm install @angular/common`

gives

```
npm ERR! Linux 3.13.0-93-generic
npm ERR! argv "/home/UX/fziacikux/.nvm/versions/node/v6.3.1/bin/node" "/home/UX/fziacikux/.nvm/versions/node/v6.3.1/bin/npm" "install" "@angular/common"
npm ERR! node v6.3.1
npm ERR! npm  v3.10.3
npm ERR! code E404

npm ERR! 404 no such package available : @angular/common
npm ERR! 404 
npm ERR! 404  '@angular/common' is not in the npm registry.

```

Log file attached.
[npm-debug.zip](https://github.com/verdaccio/verdaccio/files/444595/npm-debug.zip)

**Comment by juanpicado** [comment]
You need to configure it, take a look this thread.
https://github.com/rlidwka/sinopia/issues/399#issuecomment-222623510

**Comment by ziacik** [comment]
Thanks for that @juanpicado 
But, is this not just a workaround? Shouldn't it work out of the box?

**Comment by juanpicado** [comment]
Yeah I was wondering the same here https://github.com/verdaccio/verdaccio/issues/50 
But while verdaccio doesn't support **Scopes** I don't see should work out of the box. 
But I'd give my vote to develop scopes on verdaccio, I need it 👍

**Comment by ChadKillingsworth** [comment]
Why don't scopes work? I publish scoped packages to sinopia with no issues.

Config:

```yaml
packages:
  '@myscope/*':
    #scoped packages
    access: $all
    publish: $authenticated

  '**':
    # allow all users (including non-authenticated users) to read and
    # publish all packages
    #
    # you can specify usernames/groupnames (depending on your auth plugin)
    # and three keywords: "$all", "$anonymous", "$authenticated"
    access: $all

    # allow all known users to publish packages
    # (anyone can register by default, remember?)
    publish: $authenticated

    # if package is not available locally, proxy requests to 'npmjs' registry
    proxy: npmjs
```

**Comment by samjetski** [comment]
Locally published scoped packages work fine, but by default Sinopia/Verdaccio fails to correctly *proxy* scoped packages published on NPM. I don't think this was supported by NPM when Sinopia first implemented the feature, so the default config assumed that all scoped packages were local, not proxied.

The OOB config needs to be modified to "just work" for proxying these packages, as scoping is now being used by major projects such as `@angular`.

**Comment by ChadKillingsworth** [comment]
Support for proxying scoped packages is a trivial config change (note the double `**` in my sample above). I can PR that.

**Comment by nwhite89** [comment]
still an issue for me and @ChadKillingsworth I have tried your config this doesn't fix the issue for me

**Comment by trentearl** [comment]
This is fixed in 2.1.0
